plugins/item_tasks/web_client/routes.js

```diff
@@ -17,19 +18,50 @@ router.route('item_tasks', 'itemTaskList', () => {
 });
 
 router.route('item_task/:id/run', (id, params) => {
-    const item = new ItemModel({_id: id});
+    const itemTask = new ItemModel({_id: id});
     let job = null;
-    const promises = [item.fetch()];
+    let item = null;
+    const promises = [itemTask.fetch()];
 
     if (params.fromJob) {
         job = new JobModel({_id: params.fromJob});
         promises.push(job.fetch());
     }
 
-    $.when.apply($, promises).done(() => {
+    if (params.itemId) {
+        item = new ItemModel({_id: params.itemId});
+        promises.push(item.fetch());
+    }
+
+    $.when(...promises).done(() => {
+        let initialValues = {};
+
+        if (params.fromJob && job.has('itemTaskBindings')) {
+            initialValues = job.get('itemTaskBindings');
+        }
+
+        if (params.itemId) {
+            let prefilledItemId = params.itemId;
+            let itemTaskSpec = itemTask.get('meta').itemTaskSpec;
```

Comment:
This assumes the route's `id` points at an item that's also an item_task. The code may have made this assumption before, but I wonder if we should validate it.